server/tso/allocator_manager.go

```diff
@@ -135,38 +137,21 @@ func (am *AllocatorManager) SetLocalTSOConfig(localTSOConfig config.LocalTSOConf
 			zap.String("server-id", serverID))
 		return errs.ErrEtcdTxn.FastGenByArgs()
 	}
+	am.ClusterDCLocationChecker()
```

Comment:
If it does not load successfully, will following requests return error instead of empty map?

Reply:
Updated [here](https://github.com/tikv/pd/pull/3075/files#diff-eb404b45d0a569b05f58f4e4bf2b42b32150be75ec2ed72f86330c4b5d739eabR937).